### PR TITLE
Add aria-pressed to the editor toolbar toggle buttons.

### LINKED ISSUE #628: Improve editor toolbar toggle buttons accessibility
As noted in #558, the buttons in the inline toolbars behave as a toggle. Not just the alignment ones, also bold, italic, etc. and all the ones that need to be toggled to change a state.

Visually, their state is communicated with a color inversion. Semantically, there's nothing to communicate their state. This kind of toggle buttons need an `aria-pressed` attribute with `true/false` value. This way, assistive technologies can communicate the buttons state.

As they work now:

bold

Compare for example to what TinyMCE does:

screen shot 2017-05-03 at 17 01 39

TinyMCE uses divs for the buttons and then "resets" the real buttons semantic with `role=ppresentation`, but that's not the point, the necessary semantics is all there on the divs.

The editor toolbar would need further a11y improvements that should go in separate issues:
- first of all, a simple, easily discoverable way to focus them using only the keyboard 🙂 
- arrow key navigation as currently discussed for the inserter, see #515
- to discuss with the a11y team: maybe ARIA roles

### PR DESCRIPTION
**Pull request by afercia**
Fixes #628

### CODE REVIEW

File: editor/components/toolbar/index.js [hunk 1]
```diff
@@ -28,7 +28,9 @@ function Toolbar( { controls } ) {
 					} }
 					className={ classNames( 'editor-toolbar__control', {
 						'is-active': control.isActive
-					} ) } />
+					} ) }
+					aria-pressed={ control.isActive ? 'true' : 'false' }
```
Comment (nylen) [inline editor/components/toolbar/index.js]: I think this should just be `{ control.isActive }`.

**Comment by afercia** [comment]
Now that I look back at that, I think the original intent was to have `true` and `false` as strings, because HTML attributes are strings. See also [the aria-expanded for the inserter toggle](https://github.com/WordPress/gutenberg/blob/5bd035de02968f25d169c4702d37c35ba3f45008/editor/components/inserter/index.js#L54). Minor point though.

**Comment by nylen** [comment]
React should handle this for us. It's worth checking what it's actually doing in the dom though. Is the absence of an `aria-pressed` attribute the same as `false`?

**Comment by afercia** [comment]
>  Is the absence of an aria-pressed attribute the same as false?

No because the presence of the attribute (even when false) make screen readers announce the button as `toggle button` instead of just `button`.

**Review by aduth — APPROVED** [review]
Looks good 👍 

aria-pressed

**Comment by nylen** [comment]
Thanks for checking, @aduth, I made those last few comments while mobile.

It would be good to circle back to this later and add tests to make sure we're rendering the attribute appropriately (see https://github.com/WordPress/gutenberg/issues/641#issuecomment-299207529).

For now, +1 for merging.

**Comment by jasmussen** [comment]
I see approval here, so I'm merging.

**Pull request merged**